site/_apps/ember-hn.txt

```diff
@@ -0,0 +1,25 @@
+---
+weight: 10
+title: 'HackerNew.io'
+github-title: 'ivanvanderbyl/ember-hackernews-pwa'
+libraries:
+    - name: 'Ember.js'
+module-bundling: 'Broccoli'
+service-worker: 'Application Shell + data caching'
+perfomance-patterns: 'Asynchronous Partial Outputs'
+server-side-rendering: 'No'
+api: 'Node-hnapi (unofficial)'
+hosting: 'Cloudfront'
+authors:
+    - name: 'Ivan'
+lighthouse: '91/100'
+interactive-em: '3.1s'
+interactive-faster-3g: '2.0s'
+lighthouse-link:
+wpt-em-link: https://www.webpagetest.org/result/170511_AW_b2e2360d4207831ece22ab4210ab9444/
+wpt-faster-3g-link: https://www.webpagetest.org/result/170511_7Q_15882b2c52ed51c614ec707736e52940/
```

Comment:
Fresh run on faster 3G: 
https://www.webpagetest.org/result/170607_SC_e59cf850f6e95cf03643cc66cfe50bdb/ 7,430ms (7.4s) for first interactive. 

If we take a look at the filmstrip what we see is that there isn't any content rendered up until around the 7s or so point: https://www.webpagetest.org/video/compare.php?tests=170607_SC_e59cf850f6e95cf03643cc66cfe50bdb-r%3A1-c%3A0&thumbSize=200&ival=500&end=full